src/Framework/EventFile.cxx

```diff
@@ -58,31 +56,32 @@ EventFile::EventFile(const std::string &filename, EventFile *parent,
                                        "' is not readable or does not exist.");
     }
 
-    tree_ =
-        (TTree *)(file_->Get(ldmx::EventConstants::EVENT_TREE_NAME.c_str()));
+    // Get the tree name from the configuration
+    auto tree_name{params.getParameter<std::string>("tree_name")};
+    tree_ = static_cast<TTree *>(file_->Get(tree_name.c_str()));
```

Comment:
Hmmm... To me, the name of the event tree should be more static than python-configurable. I'm thinking it could be configured at cmake-time like the Python module name. Maybe define the event tree name global constat in `Version.h`?

Reply:
As long as there is a reasonable tree name set in ldmxcfg, I don't see an issue with this. Nobody really touches the defaults there so it's essentially the same as putting it in Version.h or in the CMake file.